ironfish/src/consensus/verifier.ts

```diff
@@ -176,27 +168,12 @@ export class Verifier {
       throw new Error('Transaction cannot deserialize')
     }
```

Comment:
This is a memory leak now, if you look here were not using `returnReference()` if no exception is thrown. We could move this to a finally clause and remove it in the catch()

Reply:
Fixed